src/engine/SCons/Tool/install.xml

```diff
@@ -46,12 +46,48 @@ which must be a directory.
 The names of the specified source files or directories
 remain the same within the destination directory. The
 sources may be given as a string or as a node returned by
-a builder.
+a builder. &Install; is accumulative, additional calls
```

Comment:
Should this be cumulative? or perhaps just simpler language?

Reply:
"Accumulative lays more emphasis on the process of accumulating, and cumulative on the result."  However, I'm not fashed either way, happy to change.  How else simpler?

Reply:
I'm thinking if you're not EFL (english as first language) accumulative is gonna be outside the vocab..